app.py

```
@@ -255,6 +285,122 @@ def toc():
+                distinct_id,
+                event='$ai_generation',
+                properties={
+                    '$ai_model': 'gpt-4o-mini',
+                    '$ai_provider': 'openai',
+                    '$ai_input': user_message,
+                    '$ai_output': ai_text,
+                    '$ai_trace_id': conversation_id,
+                }
+            )
+        except Exception:
+            pass
```

Ruff (0.12.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)